### DISCUSSION TITLE
Offline Rendering

### DISCUSSION THREAD

**Discussion by Niubbo** [apohl79/audiogridder#1128] [Ideas]
I noticed AG do not support offline rendering. Some plugins offer offline rendering options, usually it's about oversampling. It's a very useful feature to avoid the cpu usage in real time, but having a good quality when rendering.

You can take a look at Standard Clip that offer this feature (online and offline rendering options)

**Comment by apohl79** [comment]
created a feature request. thx for the suggestion!
